### PR TITLE
Add awaiting reiview label and assignees

### PR DESCRIPTION
**Pull request by SimonLab**
## merge after #111
see #62 
If a reviewer is added to the PR then the label "awaiting-review" and the reviewer is assigned to the PR.

### CODE REVIEW

**Review by samhstn — APPROVED** [review]
Perhaps look at abstracting some of the repeated code

Looks good

**Review by nelsonic — APPROVED** [review]
@SimonLab great addition. 👍 
agree with @shouston3 on abstracting. not "urgent" though, so merging this now. ✅

File: web/controllers/rules/pr/no_assignee.ex [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  @moduledoc """
+  Check for error when an PR has awaiting-review but no assignees
+  """
+  alias Dwylbot.Rules.Helpers
+  @github_api Application.get_env(:dwylbot, :github_api)
+
+  def apply?(payload) do
+    (payload["action"] == "labeled")
+    || (payload["action"] == "unassigned")
+    || (payload["action"] == "assigned")
```
Comment (samhstn) [inline web/controllers/rules/pr/no_assignee.ex:11]:
```elixir
def apply?(payload),
  do: payload["action"] in ~w(labeled unassigned assigned)
```

Comment (nelsonic) [reply web/controllers/rules/pr/no_assignee.ex:11]: next PR. 👍

**Pull request merged**